balmbees/dynamo-typeorm-stream
# Update dependency chai to v4.3.4

This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [chai](http://chaijs.com) ([source](https://togithub.com/chaijs/chai)) | devDependencies | minor | [`4.2.0` -> `4.3.4`](https://renovatebot.com/diffs/npm/chai/4.2.0/4.3.4) |
| [@types/chai](https://togithub.com/DefinitelyTyped/DefinitelyTyped) | devDependencies | patch | [`4.2.11` -> `4.2.16`](https://renovatebot.com/diffs/npm/@types%2fchai/4.2.11/4.2.16) |

---

### Release Notes

<details>
<summary>chaijs/chai</summary>

### [`v4.3.4`](https://togithub.com/chaijs/chai/releases/v4.3.4)

[Compare Source](https://togithub.com/chaijs/chai/compare/4.3.3...v4.3.4)

This fixes broken inspect behavior with bigints ([#&#8203;1321](https://togithub.com/chaijs/chai/issues/1321)) ([#&#8203;1383](https://togithub.com/chaijs/chai/issues/1383)) thanks [@&#8203;vapier](https://togithub.com/vapier)

### [`v4.3.3`](https://togithub.com/chaijs/chai/releases/4.3.3)

[Compare Source](https://togithub.com/chaijs/chai/compare/4.3.2...4.3.3)

This reintroduces `Assertion` as an export in the mjs file. See [#&#8203;1378](https://togithub.com/chaijs/chai/pull/1378) & [#&#8203;1375](https://togithub.com/chaijs/chai/issues/1375)

### [`v4.3.2`](https://togithub.com/chaijs/chai/releases/4.3.2)

[Compare Source](https://togithub.com/chaijs/chai/compare/4.3.1...4.3.2)

This fixes a regression in IE11. See [#&#8203;1380](https://togithub.com/chaijs/chai/pull/1380) & [#&#8203;1379](https://togithub.com/chaijs/chai/issues/1379)

### [`v4.3.1`](https://togithub.com/chaijs/chai/releases/4.3.1)

[Compare Source](https://togithub.com/chaijs/chai/compare/4.3.0...4.3.1)

This releases fixed an engine incompatibility with 4.3.0

The 4.x.x series of releases will be compatible with Node 4.0. Please report any errors found in Node 4 as bugs, and they will be fixed. 

The 5.x.x series, when released, will drop support for Node 4.0

This fix also ensures `pathval` is updated to `1.1.1` to fix CVE-2020-7751

### [`v4.3.0`](https://togithub.com/chaijs/chai/releases/4.3.0)

[Compare Source](https://togithub.com/chaijs/chai/compare/4.2.0...4.3.0)

This is a minor release.

Not many changes have got in since the last release but this one contains a very important change ([#&#8203;1257](https://togithub.com/chaijs/chai/issues/1257)) which will allow `jest` users to get better diffs. From this release onwards, `jest` users will be able to see which operator was used in their diffs. [The `operator` is a property of the `AssertionError` thrown when assertions fail](https://nodejs.org/api/assert.html#assert_assert_deepstrictequal_actual_expected_message). This flag indicates what kind of comparison was made.

This is also an important change for plugin maintainers. Plugin maintainers will now have access to the `operator` `flag, which they can have access to through an`util`method called`getOperator\`.

Thanks to all the amazing people that contributed to this release.

### New Features

-   Allow `contain.oneOf` to take an array of possible values ([@&#8203;voliva](https://togithub.com/voliva))
-   Adding operator attribute to assertion error ([#&#8203;1257](https://togithub.com/chaijs/chai/issues/1257)) ([@&#8203;rpgeeganage](https://togithub.com/rpgeeganage))
-   The `closeTo` error message will now inform the user when a `delta` is required ([@&#8203;eouw0o83hf](https://togithub.com/eouw0o83hf)) 

### Docs

-   Add contains flag to oneOf documentation ([@&#8203;voliva](https://togithub.com/voliva))

### Tests

-   Make sure that `useProxy` config is checked in `overwriteProperty` ([@&#8203;vieiralucas](https://togithub.com/vieiralucas))
-   Add tests for `contain.oneOf` ([@&#8203;voliva](https://togithub.com/voliva) )

### Chores

-   Update mocha to version 6.1.4
-   Add node v10 and v12 to ci ([@&#8203;vieiralucas](https://togithub.com/vieiralucas))
-   Drop support for node v4, v6 and v9 ([@&#8203;vieiralucas](https://togithub.com/vieiralucas))
-   Fix sauce config for headless chrome ([@&#8203;meeber](https://togithub.com/meeber))
-   Update dev dependencies ([@&#8203;meeber](https://togithub.com/meeber))
-   Removed phantomjs dependency ([#&#8203;1204](https://togithub.com/chaijs/chai/issues/1204))

</details>

---

### Configuration

📅 **Schedule**: At any time (no schedule defined).

🚦 **Automerge**: Enabled.

♻️ **Rebasing**: Whenever PR is behind base branch, or you tick the rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about these updates again.

---

 - [ ] If you want to rebase/retry this PR, check this box.

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).
